src/system/Wizard/WizardStep.tsx

```diff
@@ -117,7 +136,7 @@ const WizardStep = React.forwardRef(
 							 * we are adding the composed title here so that it's announced correctly by the voiceover
 							 * Using tags inside the heading would make the voiceover read the heading in multiple parts
 							 **/
-							`${ stepText }: ${ title }. ${ statusText }`
+							`${ stepText }: ${ title?.toString() }. ${ statusText }`
```

Comment:
seems like title should be a required prop

Reply:
ReactNode also includes `| string` so we can remove that too.

Reply:
changed in https://github.com/Automattic/vip-design-system/pull/251/commits/deb583d5edd10ec96d0a647cc46820b62274daa7